tests/integration/interface/allocator_c_tests.cpp

```diff
@@ -498,3 +498,21 @@ TEST(Allocators, GetInvalidId)
   ASSERT_EQ(0xDEADBEE, id);
   ASSERT_EQ(id, cpp_id);
 }
+
+TEST(Allocators, RegisterAllocation)
+{
+  umpire_resourcemanager rm;
+  umpire_allocator allocator;
+  umpire_resourcemanager_get_instance(&rm);
+  umpire_resourcemanager_get_allocator_by_name(&rm, "HOST", &allocator);
+
+  double d = 1.0;
+
+  ASSERT_NO_THROW(
+    umpire_resourcemanager_register_allocation(&rm, &d, sizeof(double), allocator)
+  );
+
+  ASSERT_NO_THROW(
+    umpire_resourcemanager_deregister_allocation(&rm, &d)
+  );
```

Comment:
My only question is whether or not we want to test this more than "can it register something" and "can it deregister something" - like, I think the c++ test also asks "given that it's registered something, can it do a simple query on it"... you know what i mean? should that kind of thing be added here? https://github.com/LLNL/Umpire/blob/a11edc3d90321990a3175a42b9932d73834f96b0/tests/integration/allocator_integration_tests.cpp#L185

Reply:
Fixed in latest commit.